sos/plugins/ipa.py

```diff
@@ -48,10 +48,11 @@ def setup(self):
         self.add_copy_spec([
             "/var/log/ipaupgrade.log",
             "/var/log/krb5kdc.log",
-            "/var/log/pki-ca/debug",
-            "/var/log/pki-ca/catalina.out",
-            "/var/log/pki-ca/system",
-            "/var/log/pki-ca/transactions",
+            "/var/log/pki/pki-tomcat/ca/debug",
+            "/var/log/pki/pki-tomcat/ca/system",
+            "/var/log/pki/pki-tomcat/ca/transactions",
+            "/var/log/pki/pki-tomcat/catalina.*"
+            "/var/log/pki/pki-ca-spawn.*"
```

Comment:
From the commit message this does appear version-dependent; we need to support current versions (at least) as far back as RHEL6 as `sos` is supported on a range of distribution release versions.